robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -103,18 +103,18 @@ def create_lookup_field(self,field_name, related):
 												   message="Detail page did not load in 1 min")
 	
 	@capture_screenshot_on_error
-	def create_custom_field(self, type, field_name, related_to=None, formula=None):
+	def create_custom_field(self, **kwargs):
 		"""Ensure that the custom field does not exist prior and Creates a custom field based on type paramenter and the field_name
```

Comment:
Should this be "based on Field_Type parameter" ??

Reply:
May also mention the acceptable keys as they have to be specifically in certain format

Reply:
can you name it starting with _ as it is a helper function